src/components/full-article-card.jsx

```diff
@@ -0,0 +1,58 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchSingleArticle } from "../utils/API";
+import { Link } from "react-router-dom";
+
+function FullArticleCard(props) {
+    
+    const {isLoading, setIsLoading} = props
+    const { article_id } = useParams();
+    console.log(article_id)
+    const [singleArticle, setSingleArticle] = useState({});
+    useEffect(() => {
+      setIsLoading(true);
+      fetchSingleArticle(article_id).then((article) => {
+        console.log(article)
+        setSingleArticle(article);
+        setIsLoading(false);
+      });
+    }, []);
+
+    console.log(singleArticle.article_img_url)
+   
+  
+    const renderArticle= (singleArticle) => {
+        return (
+        <div>
+            <main className="main-article-section">
+        <h3 className = "article-title" >{singleArticle.title}</h3>
+        <h5 className="topic"> {singleArticle.topic}</h5>
+        <p className = "article-body"> {singleArticle.body}</p>
+        <img className = "article-image" src={singleArticle.article_img_url} alt=""></img>
+       </main>
+       <section className= "article-info-section">
+          <ul className="article-info">
+            <li>By {singleArticle.author}</li>
+            <li>Written {singleArticle.created_at}</li>
+            <li>Number of Votes: {singleArticle.votes}</li>
+          </ul>
+       </section>
+       <footer>
+        <button> <Link to={'/comments'}>Read Comments</Link></button>
```

Comment:
You don't need this comments link yet because it's part of the next ticket and you're not yet showing the comments. Just stick to the content of the ticket and keep it lazer focused!
Since this isn't completed yet, it shouldn't really be a part of this pr.